testing/EstNumJellies.py

```diff
@@ -45,8 +45,12 @@ def set_land_frac_for_sugar(self, frac):
     def set_world_pop(self, people):
 
         # NE24: Add a test for type here
+	def test_type():
+		assert isinstance(people, int)
```

Comment:
This isn't quite right. You don't need to define a new function; you just need to use the assert <thing>, error syntax